src/components/text-editor/examples/text-editor.tsx

```diff
@@ -0,0 +1,31 @@
+import { LimelTextEditorCustomEvent } from '@limetech/lime-elements';
+import { Component, h, State } from '@stencil/core';
+/**
+ * A basic text editor example.
+ */
+@Component({
+    tag: 'limel-example-text-editor',
+    shadow: true,
+})
+export class BasicTextEditorExample {
+    @State()
+    private text: { html: string } = { html: '' };
+
+    public render() {
+        return [
+            <limel-text-editor onChange={this.handleChange} />,
```

Comment:
the buttons in the menu-bar need clicking twice to do anything. 
So if you want to make something bold or un-bold it, you need to click on the <kbd>B</kbd> two times.

Reply:
Is this still the case now?

Reply:
not anymore!